neuralpp/symbolic/sympy_expression.py

```diff
@@ -270,14 +293,16 @@ def internal_object_eq(self, other) -> bool:
                 return False
 
     @staticmethod
-    def from_sympy_object(sympy_object: sympy.Basic, type_dict: Dict[sympy.Basic, Expression]) -> SymPyExpression:
+    def from_sympy_object(sympy_object: sympy.Basic, type_dict: Dict[sympy.Basic, ExpressionType]) -> SymPyExpression:
         # Here we just try to find a type of expression for sympy object.
         if isinstance(sympy_object, sympy.Symbol):
             return SymPyVariable(sympy_object, type_dict[sympy_object])
         elif _is_sympy_value(sympy_object):
             return SymPyConstant(sympy_object, _infer_sympy_object_type(sympy_object, type_dict))
         elif _is_sympy_sum(sympy_object):
             return SymPySummation(sympy_object, type_dict)
+        elif _is_sympy_integral(sympy_object):
+            raise NotImplementedError("expect sympy to eliminate integral sign. TODO")
```

Comment:
We don't need a `SymPyIntegral` class for now, if we don't need to represent integral symbolically. It is also easy to add support for `SymPyIntegral`.